src/tests.builds

```diff
@@ -6,6 +6,10 @@
     <GenerateCodeCoverageReportForAll>true</GenerateCodeCoverageReportForAll>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(BuildingUAPVertical)'=='true' and '$(SkipTests)' != 'true' " >
```

Comment:
NIT: Perhaps just add a small comment in here saying why we make the projects serialized?